zoo-css-challenge/style.css

```
@@ -166,18 +182,31 @@ footer {
   background-color: rgba(255, 255, 255, 0.75);
   padding: 50px;
   max-width: 400px;
+  border-radius: 3%;
+}
+.education button{
+  font-size: 1.2em;
+}
+.education button:hover{
+  background-color: rgb(101, 2, 114);
+}
+.education button:focus{
+  background-color: rgb(101, 2, 114);
```

```suggestion
.education button:hover,
.education button:focus{
  background-color: rgb(101, 2, 114);
```
As the two selectors have the same CSS declaration, we can combine them into one rule